submissions/youhymuk/html-css-popup/index.html

```diff
@@ -0,0 +1,189 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hooli</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles/styles.css">
+</head>
+<body>
+<nav class="nav">
+  <ul class="toolbar">
+    <li class="toolbar-item">
+      <a class="mail link" href="https://mail.google.com/mail/u/0/?tab=wm&ogbl" about="_blank">
```

Comment:
What is the attribute `about` here? Maybe do you mean `target` here?

Reply:
Of course, thanks:)